### PR TITLE
status endpoint and integration of codeception

### PR DESCRIPTION
**Pull request by titopixelfusion**
- simple endpoint /api/status
- integrated codeception
- added functional test for endpoint /api/status

to run the tests, execute:
$  php vendor/bin/codecept run functional

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@
 Homestead.json
 Homestead.yaml
 .env
+
+tests/_output/*
```
Comment (jpcaparas) [inline .gitignore]: @titopixelfusion can we add these entries as well:

```
tests/_data/*
tests/_log/*
```

**Comment by jpcaparas** [comment]
@titopixelfusion 

Tests are running fine 👍 

Can we consider changing `database/db.sqlite` to `database/codeception.sqlite`? on the `.env.testing` file?

**Comment by titopixelfusion** [comment]
@jpcaparas maybe rename to db.testing.sqlite, instead of codeception, it would make more sense to have the environment name on the filename, what do you reckon?

**Comment by jpcaparas** [comment]
@titopixelfusion yep, `db.testing.sqlite` is better :)

**Pull request merged**